lumium-space/src/hooks/security/authentication/useLoginStatus.ts

```diff
@@ -2,7 +2,7 @@ import { useState } from 'react';
 import Session from 'supertokens-auth-react/recipe/session';
 
 export const useLoginStatus = (): boolean => {
-    const [loggedIn, setLoggedIn] = useState(false);
+    const [loggedIn, setLoggedIn] = useState(true);
```

Comment:
i'm not sure we should assume that by default a user is logged in... why are we doing this? why did we change this value here?

Reply:
Setting it to false will automatically route it to the signin page even tho youre logged in, this results in a loop. I made sure that no page content loads if youre not logged in, so in that case it shouldnt be an issue.

Reply:
i mean, yeah, we can merge it. who care's we're in alpha anyways